src/main/scala/tilelink/Parameters.scala

```diff
@@ -1128,13 +1128,15 @@ class TLMasterPortParameters private(
     }
   }
 
-  // Diplomatically determined operation sizes emitted by all inward Masters
+  // Diplomatically determined operation sizes emitted/supported by all inward Masters
   // as opposed to expectsVipChecker which generate circuitry to check which specific addresses
-  val allEmitClaims = masters.map(_.emits).reduce( _ intersect _)
+  val allEmitClaims    = masters.map(_.emits)   .reduce( _ intersect _)
```

Comment:
I'm not sure I see a usage case for this. Should probably remove it (and the intersect for support also).